iceoryx_binding_c/source/cpp2c_enum_translation.cpp

```diff
@@ -63,6 +63,10 @@ iox_AllocationResult AllocationResult(const iox::popo::AllocationError value)
         return AllocationResult_RUNNING_OUT_OF_CHUNKS;
     case AllocationError::TOO_MANY_CHUNKS_ALLOCATED_IN_PARALLEL:
         return AllocationResult_TOO_MANY_CHUNKS_ALLOCATED_IN_PARALLEL;
+    case AllocationError::INVALID_CHUNK:
```

Comment:
How could they get out of sync? Shouldn't we have tests to check if they are in sync?